lib/ddtrace/sampler.rb

```diff
@@ -96,6 +107,8 @@ def sample_rate(span)
 
       @mutex.synchronize do
         @sampler.fetch(key, @fallback).sample_rate
+        # TODO: set sampling metric here, for fallback case
```

Comment:
We want to set this metric anytime this sampler is used, not just for "fallback"

Reply:
👌, this was a reminder so that we don't forget to add a new metric for the existing `RateByServiceSampler`. This going to be addressed in a separate PR.